fitbenchmarking/fitbenchmarking
# 847 licence info

#### Description of Work

Fixes #847

I've changed the links to use `__` at the end, not `_`, to avoid duplicated labels warnings.

My editor has also formatted headings differently (I don't know why!) -- let me know if there is an issue and I can revert.


#### Testing Instructions

1. Check the links work and are formatted correctly
2. Is this the best way to document this information?
3. Are there any licensed plugins that we are missing?

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
